yarn.lock

```diff
@@ -229,7 +238,24 @@
     "@cucumber/messages" "^14.0.1"
     commander "^7.0.0"
 
-"@cucumber/messages@14.0.1", "@cucumber/messages@^14.0.0", "@cucumber/messages@^14.0.1":
+"@cucumber/message-streams@^1.0.0":
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/@cucumber/message-streams/-/message-streams-1.0.0.tgz#5162e65df51eb51e0aa484ef0ed46200b19a699a"
+  integrity sha512-i1Jx0EDnE+3Na82UxJ2VqE6aWWJJ+1H+3ax+SYgHmCmlUDJiJzx9dHxAAO3GISrM/RYUAuqMGHNAMnIcxkL3Pw==
+  dependencies:
+    "@cucumber/messages" "^15.0.0"
+    protobufjs "^6.10.2"
+
+"@cucumber/messages@15.0.0", "@cucumber/messages@^15.0.0":
+  version "15.0.0"
+  resolved "https://registry.yarnpkg.com/@cucumber/messages/-/messages-15.0.0.tgz#58e6541a6c21c4f0d3d4a82c24bbbfe72a6ff94c"
+  integrity sha512-LtxzSCRmYZTAKO6ucAcMflz0u90l2fev539OG+EioJ26F14KmmtxZwGabfjTxLf8NgyKeWsO8TGI2G3z4Kjr+A==
+  dependencies:
+    "@types/uuid" "^8.3.0"
+    protobufjs "^6.10.2"
+    uuid "^8.3.2"
+
+"@cucumber/messages@^14.0.1":
```

Comment:
This version still being pulled in by `html-formatter`. Shouldn't be an issue but would be good to resolve (by making a release of `html-formatter`).

Reply:
I'm stuck on that too. Looks like some issues with the build have to be fixed first.

https://github.com/cucumber/cucumber/issues/1448
https://github.com/cucumber/cucumber-jvm/pull/2268

Reply:
The release of `@cucumber/react 13.0.0` and  `@cucumber/html-formatter 13.0.0` should align all versions. I suggest we wait to merge this until those libs are released and we've upgraded this PR to use them.

Reply:
Agreed

Reply:
And updated!